.env.example

```
@@ -0,0 +1,2 @@
+GOCARDLESS_SECRET_ID={{keepass "gocardless/client_id"}}
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)